.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!--
+
+!!! IMPORTANT !!!
+
+Before creating your issue:
+
+* **Have a question?** Find community resources at [asyncapi/community](https://www.github.com/asyncapi/community).
+
+* **Found an editing mistake?** Create a Pull Request with the edited fix. The Github UI allows you to edit files directly, find the source files at [asyncapi/spec/tree/master/spec](https://www.github.com/asyncapi/spec/tree/master/spec).
+
+* **Improvements to documentation?** Head over to [asyncapi/website](https://www.github.com/asyncapi/website).
+
+* **Feature request?** First read [asyncapi/spec/blob/master/CONTRIBUTING.md](https://www.github.com/asyncapi/spec/blob/master/CONTRIBUTING.md) and prefer creating a Pull Request!
```

Comment:
where will this display and what do you mean by `and prefer creating a Pull Request!`?

Reply:
This will be displayed every time someone creates an issue that's not using an issue template, like when you click "Create blank new issue". And the "prefer a PR for a feature request" is because of this: https://github.com/asyncapi/spec/pull/511/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R104.